etc/avocado/avocado.conf

```diff
@@ -15,6 +15,8 @@ logs_dir = ~/avocado/job-results
 [sysinfo.collect]
 # Whether to collect system information during avocado jobs
 enabled = True
+# Overall timeout to collect commands, when <=0 no timeout is enforced
+# commands_timeout = 60
```

Comment:
Rather than mentioning "by default" I set the value to what reflects the default value. I hope it's clearer now...